WordPressKit/WordPressKit/SocialLogin2FANonceInfo.swift

```diff
@@ -8,4 +8,45 @@ public class SocialLogin2FANonceInfo: NSObject {
     var supportedAuthTypes = [String]() // backup|authenticator|sms
     var notificationSent = "" // none|sms
     var phoneNumber = "" // The last two digits of the phone number to which an SMS was sent.
+
+    private enum Constants {
+        static let lastUsedPlaceholder = "last_used_placeholder"
+    }
+
+    private enum AuthTypeLengths {
```

Comment:
For extra clarity it might be nice to add a comment explaining these match the server-side constants.